### PR TITLE
Add authors

### PR DESCRIPTION
**Pull request by coolalexzb**
1st PR & check permission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # pyhf-benchmark
-Benchmarking of hardware acceleration of pyhf
+
+Benchmarking of hardware acceleration of `pyhf`
+
+## Authors
+
+`pyhf-benchmark` is openly developed by [Bo Zheng](https://iris-hep.org/fellows/BoZheng.html).
+
+Please check the [contribution statistics for a list of contributors.](https://github.com/scikit-hep/pyhf/graphs/contributors)
```
Comment (matthewfeickert) [inline README.md]: The URL needs updating to be pointing at `pyhf-benchmark`
```suggestion
Please check the [contribution statistics for a list of contributors.](https://github.com/pyhf/pyhf-benchmark/graphs/contributors)
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
 # pyhf-benchmark
-Benchmarking of hardware acceleration of pyhf
+
+Benchmarking of hardware acceleration of `pyhf`
+
+## Authors
+
+`pyhf-benchmark` is openly developed by [Bo Zheng](https://iris-hep.org/fellows/BoZheng.html) and [Matthew Feickert](https://www.matthewfeickert.com/).
```
Comment (matthewfeickert) [inline README.md]:
```suggestion
`pyhf-benchmark` is openly developed by [Bo Zheng](https://iris-hep.org/fellows/BoZheng.html) and the [`pyhf` dev team](https://scikit-hep.org/pyhf/#authors).
```

I think we can put the focus on you and then just mention that we're also involved.

**Comment by matthewfeickert** [comment]
@coolalexzb Maybe also worth noting that you're an IRIS-HEP Fellow

```
## Acknowledgements

Bo Zheng was awarded an [IRIS-HEP Fellowship](https://iris-hep.org/fellows/BoZheng.html) do develop this work.
```

File: README.md (hunk 2 above)
Comment (coolalexzb) [reply README.md]: Done!

**Review by matthewfeickert — APPROVED** [review]

**Comment by coolalexzb** [comment]
I have added Acknowledgements as you mentioned. Is there anything else to add? @matthewfeickert

**Comment by coolalexzb** [comment]
I have not submitted the commit of 'Acknowledgements'

**Comment by matthewfeickert** [comment]
> Is there anything else to add?

Probably not. I just approved it to indicate that I don't have any additional comments on my side. So whenever you're happy then you can squash and merge it in.

**Comment by coolalexzb** [comment]
Everything works well! Thank you!

**Pull request merged**